blog/_posts/2017-03-01-zinc-blog-post.md

```diff
@@ -0,0 +1,141 @@
+decided to spend about a month of 1 developer's time attempting to fix these
+main blocker todo's that have prevented Greg's changes from being shipped in a
+Zinc release.
+  
+<blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr">POLL Would you like the Scala Center to finish up <a href="https://twitter.com/gkossakowski">@gkossakowski</a> improvements in Scala&#39;s incremental compiler (~40x)? <a href="https://t.co/OxKccd5hCv">https://t.co/OxKccd5hCv</a></p>&mdash; Jorge (@jvican) <a href="https://twitter.com/jvican/status/824561488962711552">26 January 2017</a></blockquote><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+  
+We're now happy to announce a new release of Zinc, ready for consumption for
+both build tools and Scala developers. While still being experimental, this is
+the first step towards getting a production-ready incremental compiler that we
+can ship in any build tool supporting Scala.
+  
+## Releasing Zinc 1.0.0-X10
+  
+The experimental release includes previous Ligthbend's improvements to the incremental
+algorithm, contributions from several OSS contributors (like [Stu Hood](https://github.com/sbt/zinc/pulls?utf8=✓&q=is:pr%20is:closed%20author:stuhood),
+[Krzysztof Romanowski](https://github.com/sbt/zinc/pulls?utf8=✓&q=is:pr%20is:closed%20author:romanowski) and
+[Peiyu Wang](https://github.com/sbt/zinc/pulls?q=is:pr+author:peiyuwang+is:closed)) and
+and the following contributions from the Scala Center:
+  
+* [Improve binary API and documentation](https://github.com/sbt/zinc/pull/236).
+* [Fix handling of As Seen From and analysis in structural types](https://github.com/sbt/zinc/pull/239).
+* [Fix handling of local dependencies for java classes](https://github.com/sbt/zinc/pull/217).
+* [Implement SafeLazy in Java](https://github.com/sbt/zinc/pull/221).
+* [Add JMH suite to benchmark any project in the
+  community and improve Zinc's performance](https://github.com/sbt/zinc/pull/225).
+* Fix [several](https://github.com/sbt/zinc/pull/21://github.com/sbt/zinc/pull/219) [infrastructure](https://github.com/sbt/zinc/pull/228) [issues](https://github.com/sbt/zinc/pull/232).
+  
+All in all, a part of these changes address all the missing tasks that [Grzegorz
+documented as missing](https://github.com/sbt/zinc/issues/174), and the rest
+improve the quality of the project. With also more insight into how to solve the
+current problems, the Scala Center has spent time figuring out how to address
+well-known issues of the incremental compiler, like [handling of changes to
+final constants](https://github.com/sbt/zinc/issues/227), [private members in
+traits](https://github.com/sbt/zinc/issues/118) and [stub errors
+handling](https://github.com/sbt/zinc/issues/150). The purpose of this work has
+been to improve even more the correctness of the compiler. In the next days, the
+Zinc team will provide an implementation for the fixes.
```

Comment:
> In the next days, the
> +Zinc team will provide an implementation for the fixes.

Not sure I follow what you mean here? From the looks of the text until here, you're listing implementation that you've already done. This is a very confusing sentence here.